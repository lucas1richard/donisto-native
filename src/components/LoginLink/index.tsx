import React from 'react';
import { View } from 'react-native';
import Txt from 'components/Txt';
import Touchable from 'components/Touchable';
import NavigationService from 'containers/Navigation/NavigationService';
import routeNames from 'containers/Navigation/routeNames';
import theme from 'theme/iftheme';

const LoginLink = () => {
  return (
    <View style={{ flexDirection: 'row', justifyContent: 'center', marginBottom: theme.fs * 2 }}>
      <Txt color="light">Already have an account? </Txt>
      <Touchable
        onPress={() => NavigationService.navigate(routeNames.LOGIN)}
      >
        <View style={{ borderBottomWidth: theme.fs * 0.05, borderBottomColor: '#fff' }}>
          <Txt color="light">Log In</Txt>
        </View>
      </Touchable>
    </View>
  );
};

export default LoginLink;
