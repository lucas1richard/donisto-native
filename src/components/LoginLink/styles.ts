import { StyleSheet } from 'react-native';
import theme from 'theme/iftheme';

export default StyleSheet.create({
  wrapper: {
    flexDirection: 'row',
    justifyContent: 'center',
    marginBottom: theme.fs * 2
  },
  loginWrapper: {
    borderBottomWidth: theme.fs * 0.05,
    borderBottomColor: '#fff'
  }
});