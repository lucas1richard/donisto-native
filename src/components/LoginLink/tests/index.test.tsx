import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginLink from '..';

describe('<LoginLink />', () => {
  const renderComponent = () => toJson(shallow(<LoginLink />));
  it('renders', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});