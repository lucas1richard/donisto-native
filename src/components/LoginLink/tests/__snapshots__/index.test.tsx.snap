// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginLink /> renders 1`] = `
<Component
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginBottom": 40,
    }
  }
>
  <Txt
    color="light"
  >
    Already have an account? 
  </Txt>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Component
      style={
        Object {
          "borderBottomColor": "#fff",
          "borderBottomWidth": 1,
        }
      }
    >
      <Txt
        color="light"
      >
        Log In
      </Txt>
    </Component>
  </TouchableOpacity>
</Component>
`;
